azext_iot/tests/central/__init__.py

```diff
@@ -43,21 +43,53 @@
 STORAGE_ACCOUNT = (
     settings.env.azext_iot_teststorageaccount or "iotstore" + generate_generic_id()[:4]
 )
+DEFAULT_FILE_UPLOAD_TTL = "PT1H"
 
 # Device templates
 device_template_path = get_context_path(__file__, "json/device_template_int_test.json")
-device_template_path_preview = get_context_path(
-    __file__, "json/device_template_int_test_preview.json"
-)
 edge_template_path_preview = get_context_path(
     __file__, "json/device_template_edge.json"
 )
 sync_command_params = get_context_path(__file__, "json/sync_command_args.json")
-device_attestation_path = get_context_path(__file__, "json/device_attestation.json")
-device_attestation2_path = get_context_path(__file__, "json/device_attestation2.json")
 device_updated_properties_path = get_context_path(__file__, "json/device_update_properties.json")
 device_updated_component_properties_path = get_context_path(__file__, "json/device_update_component_properties.json")
-DEFAULT_FILE_UPLOAD_TTL = "PT1H"
+x509_certificate_path = get_context_path(__file__, "json/x509_certificate.pem")
+
+# Device attestation
+attestation_payload = {
+    'type': 'symmetricKey',
+    'symmetricKey': {
+        'primaryKey': 'ya9+G4ED+/g0BgLduhjETJnbeEWMl1HIUApWCCpGMAU=',
```

Comment:
we have a generate_key somewhere in utility

Reply:
Used.